Source/ImPlotWidgets/Private/LineFilledGraphWidget.cpp

```diff
@@ -51,6 +45,18 @@ void SFilledGraphWidget::OnImGuiDraw() const
 	constexpr int32 GraphFlags = ImPlotFlags_AntiAliased;
 	constexpr int32 AxisFlags = ImPlotAxisFlags_NoGridLines;
 
+	TArray<float> VisibleLineMaxValues;
+
+	TArray<TArray<float>> FittedYValues;
+	for (int32 i = 0; i < YValueArrays.Num(); ++i)
+	{
+		FittedYValues.AddDefaulted();
+		for (int32 j = 0; j < YValueArrays[i].Num(); ++j)
+		{
+			FittedYValues[i].Add(FUnitConversion::Convert(YValueArrays[i][j], YUnit, BestFitUnit));
+		}
+	}
+
```

Comment:
My apologies, wouldn't even need to do this, use `SCheckBox` instead and bind to `OnCheckStateChanged`
Can discuss this more in standup tomorrow, but this will save you writing a wrapper for `SImGuiCheckBox`

Reply:
I'm going to put a separate task in for this - for all intense and purposes the one of statement will be much easier than converting this all to SCheckBox right now.